README.md

```diff
@@ -1 +1,149 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-Mobile-First] <a name="about-project"></a>
+
+**[Portfolio-Mobile-First]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Design]**
+- **[High-quality Images]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Not yet available]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- In this repository click on the "code" button (usually highlighted green) above.
+- You can select between either HTTPS or SSH if you have an SSH key linked.
+- Click on the copy icon on the right side of the link provided to copy the link
+- Go to your command line and type "git clone" followed by the link you just copied and press enter
+- The repository should be available locally on your computer.
+- You can change directory into the repository using the 'cd' command: cd "repository name"
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A code editor should be installed
+- Git should be installed
+- Nodejs should be installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:VaheStepanyan100/Portfolio-mobile-first.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [VaheStepanyan100](https://github.com/VaheStepanyan100)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[RWD]**
+- [ ] **[Preserve data in the browser]**
+- [ ] **[Validate contact]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Unfortunately, this link is broken and I suggest that you check the link to be sure that it is correct and leads to the right document ie. the license file. 👍